Library/Homebrew/cmd/create.rb

```diff
@@ -41,10 +44,12 @@ def create
 
     version = ARGV.next if ARGV.include? "--set-version"
     name = ARGV.next if ARGV.include? "--set-name"
+    tap = ARGV.next if ARGV.include? "--tap"
 
     fc = FormulaCreator.new
     fc.name = name
     fc.version = version
+    fc.tap = tap
```

Comment:
I'd probably make this `fc.tap = Tap.fetch(tap || "homebrew/core")` which would slightly simplify the logic in `FormulaCreator`. I'd also check via `Tap#installed?` that the tap is already installed and raise an exception if not.